src/gentropy/dataset/study_index.py

```diff
@@ -108,6 +108,24 @@ def get_schema(cls: type[StudyIndex]) -> StructType:
         """
         return parse_spark_schema("study_index.json")
 
+    @classmethod
+    def get_QC_column_name(cls: type[StudyIndex]) -> str:
+        """Return the name of the quality control column.
+
+        Returns:
+            str: The name of the quality control column.
+        """
+        return "qualityControls"
+
+    @classmethod
+    def get_QC_categories(cls: type[StudyIndex]) -> list[str]:
+        """Return the quality control categories.
+
+        Returns:
+            list[str]: The quality control categories.
+        """
+        return [member.value for member in StudyQualityCheck]
```

Comment:
This row had to be changed, as the previous implementation returned the attribute names (`UNRESOLVED_TARGET`) instead of the labels (`"Target/gene identifier could not match to reference."`)